### PR TITLE
Logout button

### PR DESCRIPTION
**Pull request by mouchourider**
Adding a Log Out button to the App page.

### CODE REVIEW

File: src/components/questionnarie/generalQS.jsx [hunk 1]
```diff
@@ -30,6 +31,24 @@ export default () => {
     }, 5000);
   };
 
+  const signOut = (context) => {
```
Comment (shakednai1) [inline src/components/questionnarie/generalQS.jsx:34]: you copied this section 5 times !
consider put in general place and use it from there

Comment (mouchourider) [reply src/components/questionnarie/generalQS.jsx]: Noticed it thank you. I relocate the function in Api.js.

File: src/components/SelectRoom.jsx [hunk 2]
```diff
@@ -6,14 +6,43 @@ import Container from "@material-ui/core/Container";
...
+};
 
-const SelectRoom = (props) => {
-  let history = useHistory();
   return (
+    <MyContext.Consumer>
+    {(context) => (
     <div className="ag-screen-container">
-      <h2 className="ag-header-container">בחירת חדרים</h2>
+      <h1 className="ag-header-container">בחירת חדרים{context.logged && <button
```
Comment (alongruss) [inline src/components/SelectRoom.jsx:21]: למה את מחליף את ה- h2 ל-h1?

Comment (mouchourider) [reply src/components/SelectRoom.jsx]: My bad I change it back

**Pull request merged**